llvm/test/CodeGen/AMDGPU/llvm.amdgcn.cvt.off.f32.i4.ll

```diff
@@ -0,0 +1,23 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=amdgcn -mcpu=gfx90a -verify-machineinstrs %s -o - | FileCheck %s
+; RUN: llc -mtriple=amdgcn -mcpu=gfx90a -verify-machineinstrs %s --global-isel -o - | FileCheck %s
```

Comment:
Don't use -verify-machineinstrs, move -global-isel to the front, use explicit -global-isel=0 for the dag run line. Also test with old targets too

Reply:
Added tahiti and tonga for the old targets